sound/soc/intel/boards/bridge_cs35l56.c

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: GPL-2.0-only
+//
+// Intel SOF Machine Driver with Cirrus Logic CS35L56 Smart Amp
```

Comment:
commit message

these can be used to populate 2 cs35l56 amplifiers as sidecar devices along side the cs42l43.

I am not sure about the sentence:
The cs42l43 functions as a bridge forwarding control and
audio from the SoundWire bus to these two amplifiers over SPI and I2S.

IIRC there's no control forwarding, those "sidecar" devices are directly connected to the SPI bus. The only forwarding is audio from SoundWire to I2S, isn't it?

Reply:
Yeah, I guess it depends on what one means by forwarding. Register writes over soundwire are used to control the SPI master and set the data to be sent over it, so control data goes over the soundwire and then over the SPI but it is not automatically forwarded as it were.